src/start.ts

```diff
@@ -69,12 +69,8 @@ export const handler = async (_event: any = {}): Promise<any> => {
         );
 
         // Fetch current block number from chain via provider
-        const retryableGetBlockNumberCall = node.utils.retryOnTimeout(
-          DEFAULT_RETRY_TIMEOUT_MS,
-          () => provider.getBlockNumber()
-        );
-        const [err, currentBlock] = await node.utils.go(
-          () => retryableGetBlockNumberCall
+        const [err, currentBlock] = await retryGo(() =>
```

Comment:
`const [err, currentBlock] = await retryGo(provider.getBlockNumber);`
I've had this exact case suggested to me in a PR^
It's looking good! I'm excited :)

Reply:
Yeah. I tried doing that by reading your [first comment](https://github.com/api3dao/airkeeper/pull/6#discussion_r774900426) but unfortunately didn't work as expected.

If I remove the wrapping arrow function then when `getBlockNumber` uses the keyword `this` internally I got a `not defined` error message. I was able to fix it by doing:
```const [err, currentBlock] = await retryGo(provider.getBlockNumber.bind(provider));```
but I thought it was not an improvement over just having a wrapper arrow function.

BTW your suggestion about using the retryGo() function made the code look much nicer 🔥

Reply:
Wow, yeah, interesting. I had a similar thing in my gas pricing PR which was picked up on.

> using the retryGo() function made the code look much nicer

You've done a great job :)